addon/tests/helpers/test-done.js

```diff
@@ -0,0 +1,131 @@
+import Ember from "ember";
```

Comment:
wow this is some serious code dude! also- why did you kill the ember import? looks like you use Ember :)